### ISSUE TITLE
ViT特征图尺寸和代码中输入SFPN的尺寸不匹配问题

### ISSUE THREAD

**Issue by I-AM-Not-Xx** [SunnyHaze/IML-ViT#54]
问题一：
ViT输出的尺寸是(1, 768, 64, 64)，但为什么输入SFPN的尺寸是(1, 768, 256, 256)
SFPN部分代码如下
        self.featurePyramid_net = SimpleFeaturePyramid(
            in_feature_shape= (1, embed_dim, 256, 256),
            out_channels= fpn_channels,
            scale_factors=fpn_scale_factors, #缩放因子
            top_block=LastLevelMaxPool(), #顶层块使用最大池化
            norm="LN",    
        )
问题二：
论文中SFPN模块的输出是4个尺度1/4、1/8、1/16、1/32
但是代码中给出的是五个尺度
这个代码位于iml_vit_model.py
fpn_scale_factors = (4.0, 2.0, 1.0, 0.5),
代码如下（这段代码位于modules.decoderhead.py的最后）
 def forward(self, x):
        c1, c2, c3, c4, c5 = x    # 1/4 1/8 1/16 1/32 1/64
        
        n, _ , h, w = c1.shape # Target size of all the features
        
        # _c1 = self.linear_c1(c1).reshape(shape=(n, -1, c1.shape[2], c1.shape[3]))
        
        _c1 =  F.interpolate(c1, size=(h, w), mode='bilinear', align_corners=False)
        
        # _c2 = self.linear_c2(c2).reshape(shape=(n, -1, c2.shape[2], c2.shape[3]))
        
        _c2 = F.interpolate(c2, size=(h, w), mode='bilinear', align_corners=False)
        
        # _c3 = self.linear_c3(c3).reshape(shape=(n, -1, c3.shape[2], c3.shape[3]))
        
        _c3 = F.interpolate(c3, size=(h, w), mode='bilinear', align_corners=False)
        
        # _c4 = self.linear_c4(c4).reshape(shape=(n, -1, c4.shape[2], c4.shape[3]))
        
        _c4 = F.interpolate(c4, size=(h, w), mode='bilinear', align_corners=False) 
        
        # _c5 = self.linear_c5(c5).reshape(shape=(n, -1, c5.shape[2], c5.shape[3]))
        
        _c5 = F.interpolate(c5, size=(h, w), mode='bilinear', align_corners=False)
        
        _c = self.linear_fuse(torch.cat([_c1, _c2, _c3, _c4, _c5], dim=1))
        
        _c = self.norm(_c)
        
        x = self.dropout(_c)
        
        x = self.linear_predict(x)
        
        return x
